Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/ExampleAdapter.cpp

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzFramework/DocumentPropertyEditor/AdapterBuilder.h>
+#include <AzFramework/DocumentPropertyEditor/ExampleAdapter.h>
+
+namespace AZ::DocumentPropertyEditor
+{
+    ExampleAdapter::ExampleAdapter()
```

Comment:
Should this `ExampleAdapter` live in the standalone view project instead of the DPE framework?

Reply:
Yeah good callout, I was thinking about unit testing something of the functionality it if I had time but 1) I didn't lol and 2) we could just add unit tests to the standalone project later